src/main/java/com/gtnewhorizon/gtnhlib/core/GTNHLibCore.java

```diff
@@ -82,7 +82,12 @@ public String getMixinConfig() {
 
     @Override
     public List<String> getMixins(Set<String> loadedCoreMods) {
-        return Mixins.getEarlyMixins(loadedCoreMods);
+        final List<String> mixins = Mixins.getEarlyMixins(loadedCoreMods);
+        if (Boolean.parseBoolean(System.getProperty("gtnhlib.debugtextures", "false"))) {
```

Comment:
That should be in the Mixins enum class like the rest of them with an apply condition of `.setApplyIf(() -> Boolean.getBoolean("gtnhlib.debugtextures"))`

Reply:
why ? it's a debug feature

Reply:
Why would that matter, either way it only gets applied if the jvm flag is set